releasenotes/notes/deprecate_external_ddtrace.propagation.utils-e5106df5b4723310.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    The contents of ``ddtrace.propagation.utils.py`` have been moved into ``ddtrace.propagation._utils.py`` in an
+    effort to internalize the module. Public methods have been imported back
+    into ``ddtrace.propagation.utils.py`` in order to retain compatibility, but this module will be
+    removed entirely in version 1.0.0.
```

Comment:
We can probably simplify this message to be more like:

> `ddtrace.propagation.utils.from_wsgi_header` and `ddtrace.propagation.utils.get_wsgi_header` have been deprecated and will be removed in version 1.0.0.


Or, if they really are going to `ddtrace.contrib.trace_utils`, we can just do that move now and state that as the release note.

Reply:
When I moved them to `ddtrace.contrib.trace_utils` I got a circular import error because of the propagation.http module, so Kyle just recommended I go the normal deprecation route.  I'll simplify the message!

Reply:
Actually I'd prefer we just say


> `ddtrace.propagation.utils` has been deprecated and will be removed in version 1.0.